.github/workflows/interface-unit-tests.yml

```
@@ -494,7 +494,7 @@ jobs:
       additional_pip_packages: |
         pyzx matplotlib stim quimb mitiq ply optax scipy-openblas32>=0.3.26 qualtran
         git+https://github.com/PennyLaneAI/pennylane-qiskit.git@master
-        ${{ needs.default-dependency-versions.outputs.jax-version }}
+        jax==0.4.28 jaxlib==0.4.28
```

We should probably wait for Catalyst to merge their upgrade PR before we merge this one, and revert this file accordingly. What do you think?

```suggestion
        ${{ needs.default-dependency-versions.outputs.jax-version }}
```